templates/templates/generic-component-busola.yaml

```diff
@@ -0,0 +1,127 @@
+{{- /*
+   This template renders jobs for a component using generic buildpack with make and docker
+
+   Required values:
+     - repository - repository address (e.g. github.com/kyma-project/kyma
+     - path - address of the component within the repository
+     - pushRepository - the suffix of the preset-docker-push- label
+     - bootstrapTag - tag of the bootstrap image
+     - ReleaseBranchPattern - pattern for release branch prefix (default: ^release-<supported-releases>-<component-dir-name>$)
+     - runIfChanged - a list of regexps to watch in addition to path (default: [])
+     - resources.memory - memory given to job container (default: 1.5Gi)
+     - resources.cpu - cpu given to job container (default: 0.8)
+  */ -}}
+
+{{ $repositoryName := index (.Values.repository | splitList "/") 2 -}}
+{{- $additionalRunIfChanged := .Values.additionalRunIfChanged | default (list) }}
+{{- $runIfChanged := prepend $additionalRunIfChanged (print "^" .Values.path "/") | join "|" }}
+{{- $componentName := .Values.path | splitList "/" | last }}
+{{- $isNotTestInfra := ne .Values.repository "github.com/kyma-project/test-infra"}}
+{{- $hasReleaseBounds := and (not .Values.since) (not .Values.until) }}
+{{- $releases := matchingReleases .Global.releases .Values.since .Values.until }}
+{{- $isValidForNextRelease := releaseMatches .Global.nextRelease .Values.since .Values.until }}
+{{- $releasesPattern := $releases | join "|" | replace "." "\\." }}
+{{- $releaseBranchReleasesPattern := $hasReleaseBounds | ternary ".*" (print "("  $releasesPattern ")") }}
+{{- $defaultReleaseBranchPattern := (print "^release-" $releaseBranchReleasesPattern "-" $componentName "$") }}
+{{- $defaultReleaseBranchPatternOrEmpty := ne (len $releases) 0 | ternary $defaultReleaseBranchPattern "" }}
+{{- $releaseBranchPattern := .Values.ReleaseBranchPattern | default $defaultReleaseBranchPatternOrEmpty }}
+
+{{- if $isNotTestInfra }}
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+{{- end }}
+
+job_template: &job_template
+  decorate: true
+  path_alias: {{ .Values.repository }}
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-{{ .Values.pushRepository }}: "true"
+  run_if_changed: "{{ $runIfChanged }}"
+  spec:
+    containers:
+      - image: eu.gcr.io/kyma-project/test-infra/bootstrap:{{ .Values.bootstrapTag }}
+        securityContext:
+          privileged: true
+        command:
+          - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/build-generic.sh"
+        args:
+          - "/home/prow/go/src/{{ .Values.repository }}/{{ .Values.path }}"
```

Comment:
Here. If the build stage has different name than `release`, which is the case in the [Makefile](https://github.com/kyma-project/busola/blob/main/Makefile), you have to move this `arg` list to the definition of actual prowjob.

Reply:
We have release build stage in makefiles